Packs/CommonScripts/Scripts/BMCTool/BMCTool.py

```diff
@@ -0,0 +1,502 @@
+                              192, 0, 128, 128, 224, 0, 128, 160, 0, 0, 128, 160, 32, 0, 128, 160, 64, 0, 128, 160, 96,
+                              0, 128, 160, 128, 0, 128, 160, 160, 0, 128, 160, 192, 0, 128, 160, 224, 0, 128, 192, 0, 0,
+                              128, 192, 32, 0, 128, 192, 64, 0, 128, 192, 96, 0, 128, 192, 128, 0, 128, 192, 160, 0,
+                              128, 192, 192, 0, 128, 192, 224, 0, 128, 224, 0, 0, 128, 224, 32, 0, 128, 224, 64, 0, 128,
+                              224, 96, 0, 128, 224, 128, 0, 128, 224, 160, 0, 128, 224, 192, 0, 128, 224, 224, 0, 192,
+                              0, 0, 0, 192, 0, 32, 0, 192, 0, 64, 0, 192, 0, 96, 0, 192, 0, 128, 0, 192, 0, 160, 0, 192,
+                              0, 192, 0, 192, 0, 224, 0, 192, 32, 0, 0, 192, 32, 32, 0, 192, 32, 64, 0, 192, 32, 96, 0,
+                              192, 32, 128, 0, 192, 32, 160, 0, 192, 32, 192, 0, 192, 32, 224, 0, 192, 64, 0, 0, 192,
+                              64, 32, 0, 192, 64, 64, 0, 192, 64, 96, 0, 192, 64, 128, 0, 192, 64, 160, 0, 192, 64, 192,
+                              0, 192, 64, 224, 0, 192, 96, 0, 0, 192, 96, 32, 0, 192, 96, 64, 0, 192, 96, 96, 0, 192,
+                              96, 128, 0, 192, 96, 160, 0, 192, 96, 192, 0, 192, 96, 224, 0, 192, 128, 0, 0, 192, 128,
+                              32, 0, 192, 128, 64, 0, 192, 128, 96, 0, 192, 128, 128, 0, 192, 128, 160, 0, 192, 128,
+                              192, 0, 192, 128, 224, 0, 192, 160, 0, 0, 192, 160, 32, 0, 192, 160, 64, 0, 192, 160, 96,
+                              0, 192, 160, 128, 0, 192, 160, 160, 0, 192, 160, 192, 0, 192, 160, 224, 0, 192, 192, 0, 0,
+                              192, 192, 32, 0, 192, 192, 64, 0, 192, 192, 96, 0, 192, 192, 128, 0, 192, 192, 160, 0,
+                              240, 251, 255, 0, 164, 160, 160, 0, 128, 128, 128, 0, 0, 0, 255, 0, 0, 255, 0, 0, 0, 255,
+                              255, 0, 255, 0, 0, 0, 255, 0, 255, 0, 255, 255, 0, 0, 255, 255, 255, 0)))
+    COLOR_BLACK = b"\x00"
+    COLOR_WHITE = b"\xFF"
+
+    def __init__(self, verbose=False, count=0, old=False, big=False, width=64):
+        self.bdat = ""
+        self.o_bmps = []
+        self.bmps = []
+        self.btype = None
+        self.cnt = count
+        self.fname = None
+        self.oldsave = old
+        self.pal = False
+        self.verb = verbose
+        self.big = big
+        self.STRIPE_WIDTH = width
+        if count > 0:
+            self.b_log("debug", True, f"At most {count} tiles will be processed.")
+        if old:
+            self.b_log("debug", True, "Old data will also be saved in separate files.")
+
+    def b_log(self, ltype, verbose, lmsg):
+
+        log_type = {"info": demisto.info,
+                    "debug": demisto.debug,
+                    "error": demisto.error
+                    }
+
+        if not verbose or self.verb:
```

Comment:
Don't really understand the condition, if verbose is set to False we will print the log.
Why not 
`if verbose or self.verb`?